Makefile

```diff
@@ -14,17 +14,15 @@ PARENT_IMG ?= $(shell source ci/config/env; echo $${PARENT_IMG})
 # proof of concepts.
 
 # Customizable cluster domain for deploying wherever we want
-# CLUSTER_DOMAIN ?= $(shell kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain')
-CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain'
+CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o jsonpath='{.spec.baseDomain}'
```

Comment:
as discussed during review, if `CLUSTER_DOMAIN` is not used for anything, remove all references, checks, etc.

Reply:
Updated and checked.